src/key.rs

```diff
@@ -135,11 +133,15 @@ impl Serialize for PublicKey {
 mod tests {
     use super::*;
 
-    use rand::{SeedableRng, XorShiftRng};
+    use rand::SeedableRng;
+    use rand_xorshift::XorShiftRng;
 
     #[test]
     fn test_bytes_roundtrip() {
-        let rng = &mut XorShiftRng::from_seed([0x3dbe6259, 0x8d313d76, 0x3237db17, 0xe5bc0654]);
+        let rng = &mut XorShiftRng::from_seed([
+            0x59, 0x62, 0xbe, 0x5d, 0x76, 0x3d, 0x31, 0x8d, 0x17, 0xdb, 0x37, 0x32, 0x54, 0x06,
```

Comment:
Is it intentional that the forth byte changed from 0x3d to 0x5d? At least also the verification was changed accordingly.